doc/source/install/ubuntu.md

```diff
@@ -120,3 +120,92 @@ sudo apt -V -y install groonga-normalizer-mysql
 Build from source is for developers.
 
 See {doc}`/install/cmake` .
+
+## Migration from Groonga PPA (ppa:groonga/ppa) to Groonga APT Repository (packages.groonga.org)
+
+This section guides you through the migration from the deprecated Groonga PPA
+(Personal Package Archive, ppa:groonga/ppa) to our new Groonga APT Repository
+(packages.groonga.org).
+
+By switching to the Groonga APT Repository, you'll receive packages built with
+Apache Arrow enabled. Which unlocks extra features such as parallel offline
+index building. We strongly recommend you to migrate to enjoy these enhancements.
+
+If you are currently using the Groonga PPA, please follow the
+steps below to switch to the new package source.
+
+### Configure the Groonga APT Repository
+
+#### Register Groonga APT Repository
+
+To register the Groonga APT Repository, install the `groonga-apt-source package`
+as follows.
+
+```bash
+sudo apt update
+sudo apt install -y -V groonga-apt-source
+sudo apt update
+```
+
+#### Remove the Existing Package Source
+
+Remove the old Groonga PPA:
+
+```bash
+sudo add-apt-repository -y --remove ppa:groonga/ppa
+sudo apt update
+```
+
+After these steps, your package source is now switched from Groonga PPA to
+Groonga APT Repository.
+
+### Upgrade Package
+
+There are two approaches depending on your needs.
+
+#### Option 1: Use Groonga APT Repository from the Next Release (Recommended)
+
+Starting with the next package version, we strongly recommend using the Groonga
+APT Repository. When you upgrade Groonga via `apt`, you will automatically receive
+the next package version without having to specify a version explicitly as
+follows. This option is simpler and is recommended for most users:
+
+```bash
+sudo apt upgrade package-name
```

Comment:
Do we need `package-name` here?

In general, users use `sudo apt upgrade` not `sudo apt upgrade package-name`.

Reply:
fix: [b1d84ab](https://github.com/groonga/groonga/pull/2260/commits/b1d84abe4520a5eba3a01604155a469b6485f254) Thanks. We don't need it here.